packages/hydrojudge/package.json

```diff
@@ -32,4 +32,4 @@
         "build": "tsc",
         "lint": "eslint . --ext ts --fix"
     }
-}
\ No newline at end of file
+}
```

Comment:
no \n as yarn always remove this empty line. (thats really annoying)

Reply:
my fault. that's related to vscode not yarn. (but I want to keep the current behavior.)